src/Form/Ingest/Review.php

```diff
@@ -145,7 +147,7 @@ public function submitActivation(array &$form, FormStateInterface $form_state) {
    * @param mixed $context
    *   A reference to the batch context.
    */
-  public function runBatchOp(MigrationInterface $migration, MigrateBatchExecutable $e, &$context) {
+  public function runBatchOp(MigrationInterface $migration, MigrateExecutable $e, &$context) {
```

Comment:
Changing this because both MigrateBatchExecutable and MigrationRollbackBatch extends MigrateExecutable